auslib/test/web/test_client.py

```diff
@@ -1048,9 +1048,16 @@ def testXContentTypeOptionsIsSetFor500(self):
             self.assertEqual(ret.headers.get("X-Content-Type-Options"), "nosniff")
 
     def testEmptySnippetOn404(self):
-        ret = self.client.get('/whizzybang')
+        ret = self.client.get('/update/whizzybang')
         self.assertUpdatesAreEmpty(ret)
 
+    @given(just('/whizzybang'))
+    @example(path='/')
+    @example(path='/api/v1')
```

Comment:
It would be good to include something under /api/v1/rules/ and /api/v1/releases/, too.